data/api/client-server/report_content.yaml

```
@@ -29,11 +29,20 @@ securityDefinitions:
 paths:
   "/rooms/{roomId}/report/{eventId}":
     post:
-      summary: Reports an event as inappropriate.
+      summary: Reports an event as inappropriate. You must be joined to the room in order
+      to report an event.
```

The validator be unhappy, and the summary is "wrong" from a modern point of view:
```suggestion
      summary: Report an event in a joined room as inappropriate.
```

shorter is better for the `summary`.